app/views/api/events/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+@events.each do |event|
+  json.set! event.id do
```

Comment:
Since we are working with two separate repos, I don't think we need this view folder cuz we are going to fetch the API in the react App.

Reply:
@nedjwab what is on the view files are just JSON files. I didn't build any html there

Reply:
@Stanleeeeee alright i checked it , you can merge it